src/components/MenuItem.js

```diff
@@ -22,24 +22,29 @@ const propTypes = {
     /** Text to display for the item */
     title: PropTypes.string.isRequired,
 
-    /** Boolean whether to display the ArrowRight icon */
-    shouldShowRightArrow: PropTypes.bool,
+    /** Boolean whether to display the right icon */
+    shouldShowRightIcon: PropTypes.bool,
```

Comment:
Why do we need an extra prop for this? Shouldn't the render method just look at `iconRight` and if there is no value, then it wouldn't display anything?

Reply:
The reason for having a boolean because for every usage we need to import the icon and pass that as a prop. I agree that we can do that but the majority of the use cases in the code uses an arrow icon, hence importing it every time might be tedious.

Reply:
Yeah, my original suggestion was to simply check if the icon prop had a value. 

I wouldn't block a suggestion to make this change, but upon seeing the PR it made sense to keep the arrow as default, because overriding the default icon is the exception to the rule here -- every other menu item will display the chevron and it felt redundant to specifically state this on every menu item.